controllers/kam.go

```diff
@@ -253,10 +253,10 @@ func (r *ReconcileGitopsService) reconcileCLIServer(cr *pipelinesv1alpha1.Gitops
 		}
 	}
 
-	routeRef := newRouteForCLI()
-	if err := controllerutil.SetControllerReference(cr, routeRef, r.Scheme); err != nil {
-		return reconcile.Result{}, err
-	}
+	//routeRef := newRouteForCLI()
+	// if err := controllerutil.SetControllerReference(cr, routeRef, r.Scheme); err != nil {
+	// 	return reconcile.Result{}, err
+	// }
 
 	err = r.Client.Get(context.TODO(), types.NamespacedName{Name: routeRef.Name, Namespace: routeRef.Namespace}, &routev1.Route{})
```

Comment:
@varshab1210 
Im guessing the build is failing because you've commented out `routeRef` above but it is being referenced here

Reply:
Hi @jaideepr97 
This was intentional to see if the operator bundle is picking up the changes from this PR or not. 
If the bundle created including this change, **1-001_validate_kam_service** should have failed but if you look at [this](https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/redhat-developer_gitops-operator/428/pull-ci-redhat-developer-gitops-operator-master-v4.9-kuttl-sequential/1620825300738772992) job, only **1-077_validate_workload_status_monitoring_alert** has failed, which is not the expected behavior.

Reply:
I don't it would reach bundle creation stage if the code itself is not compiled correctly

Reply:
But Kuttl test requires the bundle to be created and even tests are passing. That means its not taking into account whatever changes have been made in this PR